workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityWorkerMenuGenerator.tt

```diff
@@ -0,0 +1,84 @@
+﻿<#@ template language="C#" #>
+<#@ output extension=".cs" #>
+<#
+    var generatedHeader = CommonGeneratorUtils.GetGeneratedHeader();
+    var workerTypeList = GetWorkerTypes();
+#>
+<#= generatedHeader #>
+
+#if UNITY_EDITOR
+using Improbable.Gdk.BuildSystem;
+using Improbable.Gdk.BuildSystem.Configuration;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace <#= qualifiedNamespace #>
+{ 
+    internal static class BuildWorkerMenu
+    {
+        private const string LocalMenu = "Build for local";
+        private const string CloudMenu = "Build for cloud";
+
+        private static readonly string[] AllWorkers = 
+        {
+<# foreach (var workerType in workerTypeList) { #>
+            "<#= workerType #>",
+<# } #>
+        };
+
+<# 
+for (var i = 0; i < workerTypeList.Count; i++) { 
+var workerType = workerTypeList[i];
+#>
+        [MenuItem(EditorConfig.ParentMenu + "/" + LocalMenu + "/<#= workerType #>", false, <#=i #>)]
```

Comment:
Nit: `<#= i #>`. Also do we want to offset the menu so it appears in a particular place?

Reply:
Yes, the order they were in previously was decided after a LOT of discussion with Beth. We should stick to something similar